pkg/operator/ceph/file/filesystem.go

```diff
@@ -180,6 +179,13 @@ func (f *Filesystem) doFilesystemCreate(context *clusterd.Context, cephVersion c
 					fmt.Sprintf(": %+v", err),
 			)
 		}
+		if err := client.SetPools(context, cephVersion, clusterName, f.Name, f.dataPools, f.metadataPool); err != nil {
+			logger.Errorf(
+				fmt.Sprintf("failed to change replicated size for dataPools to %d for filesystem %s, and replicated size for metadataPool to %d for filesystem %s still continuing. ", f.dataPools[0].ReplicatedConfig.Size, f.Name, f.metadataPool.ReplicatedConfig.Size, f.Name) +
```

Comment:
This error should be simple and aggregate the error from the return of `SetPools()`. For example:
```
logger.Errorf("failed to set pool size. %+v", err)
```

Reply:
I tried taking the reference from the above function, so I tried to incorporate as much info as possible.
I’ll make changes as requested.

Reply:
why there is a hardcoded index with no length checks? A hardcoded index can throw IndexOutOfBound error in runtime.

Reply:
Thanks for the tip!
Added a `for` loop instead.